tests/responses/t2k/test_1.html

```diff
@@ -0,0 +1,93 @@
+
+
+
+<!DOCTYPE html>
+
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-us" lang="en-us">
+
+
+
+<title>T2K Theses (T2K-THESIS-nnn) &mdash; T2K Intranet</title>
+<body>
+</div>
+
+
+
+
+                <h1 class="documentFirstHeading">T2K Theses (T2K-THESIS-nnn)</h1>
+		<h4><a href="drawcatview">Switch to Catagorized View</a></h4>
+
+		<div><p>T2K theses are those prepared by graduate students in the T2K collaboration to complete their degrees. The final version should be <strong>submitted by the student</strong>.</p></div>
+
+<table border="1" cellpadding="2" cellspacing="2" class="grid listing" id="folders">
+    <thead>
+        <span>
+            <th>
+                <span>Number</span>
+            </th>
+        </span>
+        <span>
+            <th>
+                <span>Name</span>
+            </th>
+        </span>
+        <span>
+            <th>
+                <span>Title</span>
+            </th>
+        </span>
+        <span>
+            <th>
+                <span>Folder Created</span>
+            </th>
+        </span>
+    </thead>
+    <tbody>
+        <span>
+            <tr>
+		<span>
+		   <td>
+                       <span>
+                           <a href="001.html">
```

Comment:
@bittirousku Here is perhaps why it is getting the 404 probably as it takes this link and adds `001.html`?